DQM/L1TMonitor/src/L1TStage2CPPF.cc

```diff
@@ -0,0 +1,138 @@
+/*
+ * \L1TStage2CPPF.cc
+ * \author
+ * \September 2018
+*/
+
+#include "DQM/L1TMonitor/interface/L1TStage2CPPF.h"
+
+L1TStage2CPPF::L1TStage2CPPF(const edm::ParameterSet& ps)
+    : monitorDir(ps.getUntrackedParameter<std::string>("monitorDir", "")),
+      //  cppfdigiLabel(ps.getParameter<edm::InputTag>("cppfdigiLabel")),
+      verbose(ps.getUntrackedParameter<bool>("verbose", false)),
+      global_phi(-1000),
+      cppfDigiToken_(consumes<l1t::CPPFDigiCollection>(ps.getParameter<edm::InputTag>("cppfSource"))) {}
+
+L1TStage2CPPF::~L1TStage2CPPF() {}
+
+void L1TStage2CPPF::bookHistograms(DQMStore::IBooker& ibooker, const edm::Run& iRun, const edm::EventSetup& eveSetup) {
+  ibooker.setCurrentFolder(monitorDir);
+
+  Occupancy_EMTFSector = ibooker.book2D("Occupancy_EMTFSector", "Occupancy_EMTFSector", 36, 1., 37., 12, 1., 13.);
+  Track_Bx = ibooker.book2D("Track_Bx", "Track_Bx", 12, 1., 13., 7, -3., 4.);
+}
+
+void L1TStage2CPPF::analyze(const edm::Event& eve, const edm::EventSetup& eveSetup) {
+  if (verbose) {
+    edm::LogWarning("L1TStage2CPPF") << "L1TStage2CPPF: analyze...." << std::endl;
+  }
+
+  edm::Handle<l1t::CPPFDigiCollection> CppfDigis;
+  eve.getByToken(cppfDigiToken_, CppfDigis);
+
+  //Fill the specific bin for each EMTF sector
+  for (int i = 1; i < 7; i++) {
+    EMTFsector1bins.push_back(i);
+    EMTFsector2bins.push_back(i + 6);
+    EMTFsector3bins.push_back(i + 12);
+    EMTFsector4bins.push_back(i + 18);
+    EMTFsector5bins.push_back(i + 24);
+    EMTFsector6bins.push_back(i + 30);
+  }
+  //FIll the map for each EMTF sector
+  fill_info[1] = EMTFsector1bins;
+  fill_info[2] = EMTFsector2bins;
+  fill_info[3] = EMTFsector3bins;
+  fill_info[4] = EMTFsector4bins;
+  fill_info[5] = EMTFsector5bins;
+  fill_info[6] = EMTFsector6bins;
+
+  for (auto& cppf_digis : *CppfDigis) {
+    RPCDetId rpcId = cppf_digis.rpcId();
+    int ring = rpcId.ring();
+    int station = rpcId.station();
+    int region = rpcId.region();
+    int subsector = rpcId.subsector();
+
+    //TOBECHECKED why rpcId.sector() sometimes return 7 ?
+    if (cppf_digis.emtf_sector() != -99) {
+      std::cout << " L1TStage2CPPF::analyze cppf_digis.emtf_sector=" << cppf_digis.emtf_sector() << std::endl;
```

Comment:
There are still a few occurrences of couts, like here. Could you please check everywhere in the new code?

Reply:
Hi @cecilecaillol , many thinks for your quick reply. Please check. There were two couts here, which now have been removed. Thanks.